lib/l10n_utils/dotlang.py

```diff
@@ -108,7 +105,7 @@ def translate(text, files):
     if lang == settings.LANGUAGE_CODE:
         return Markup(text)
 
-    tweaked_text = tweak_message(text)
+    tweaked_text = strip_whitespace(text)
```

Comment:
Oh, nm, this is on purpose. `tweak_message` is specifically for the extract stuff because it has handles for the pluralization machinery. If you pass just a string to `tweak_message` it just calls `strip_whitespace` and returns, so these are functionally identical and this way is one fewer function call and a bit more straightforward code path.